docs/nodes/light-node.md

```diff
@@ -69,6 +69,17 @@ celestia light start --core.grpc <ip>:9090
 
 If you need a list of RPC endpoints to connect to, you can check from the list [here](../nodes/mamaki-testnet#rpc-endpoints)
 
+Once you start the Light Node, a wallet key will be generated for you.
+You will need to fund that address with some tokens to pay for PayForData transactions.
+You can find the address by:
+
+1. Checking the logs output
+2. Using the following command:
+
+```sh
+./cel-key list --node.type light --keyring-backend test
```

Comment:
The command `./cel-key` doesn't exist anywhere from the previous steps.

Reply:
right, the whole guide, and for the other 2 nodes too, is missing that and/or the link to the keys page, should we maybe link the keys page first then? cc @YazzyYaz